src/lib/mina_base/zkapp_command.ml

```diff
@@ -1955,6 +1955,14 @@ let has_zero_vesting_period t =
       | Set { vesting_period; _ } ->
           Mina_numbers.Global_slot_span.(equal zero) vesting_period )
 
+let is_incompatible_version t =
+  Call_forest.exists t.account_updates ~f:(fun p ->
+      match p.body.update.permissions with
+      | Keep ->
+          false
+      | Set { set_verification_key = auth, txn_version; _ } ->
```

Comment:
Do you not get a warning for this unused `auth` value? If so, we should work to re-enable the unused value warnings for this file (in a separate PR).